### DISCUSSION TITLE
I would like to konw where  I can download the PK-Sim software

### DISCUSSION THREAD

**Discussion by Dyw199072** [Open-Systems-Pharmacology/Forum#1770] [Feature Requests & Ideas]
Hi everyone, I am a graduate student and would like to download PK-Sim software for my studies, where can I download the full version of this software? I would appreciate if you could also tell me how to install it! Thank you all~

**Comment by JanSchlender** [comment]
Hi, have a look here:
https://github.com/Open-Systems-Pharmacology/Suite/releases/tag/v11.3

**Reply by Dyw199072** [reply]
Dear Friend,
I followed the link you gave me to try to download the software, but I was prompted that it could not be downloaded and needed authorization. What should I do if I encounter this problem?

**Comment by Dyw199072** [comment]
Thank you very much for your help, I'm going to try it!




Michael'vara
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "Open-Systems-Pharmacology/Forum"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年6月2日(星期天) 晚上9:37
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [Open-Systems-Pharmacology/Forum] I would like to konw where  I can download the PK-Sim software (Discussion #1770)





 
Hi, have a look here:
 https://github.com/Open-Systems-Pharmacology/Suite/releases/tag/v11.3
